bindings/azure/blobstorage/blobstorage.go

```diff
@@ -138,25 +138,21 @@ func (a *AzureBlobStorage) create(blobURL azblob.BlockBlobURL, req *bindings.Inv
 		delete(req.Metadata, cacheControl)
 	}
 
-	// Unescape data which will still be a JSON string
-	unescapedData, unescapeError := strconv.Unquote(string(req.Data))
-
-	if unescapeError != nil {
-		return nil, unescapeError
+	d, err := strconv.Unquote(string(req.Data))
```

Comment:
If json serializer serializes `string` data, it adds double-quotes `"string"`. If we want to save only `string`, not a json-serialized one, then this is correct. is this what we want ?

Reply:
Yes. If a JSON string is sent, it's saved as is. I validated it.